rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -1200,17 +1201,24 @@ def get_multiple_erc20_contract_info(self, addresses: list[ChecksumEvmAddress])
 
             for idx, single_output in enumerate(get_chunks(output, ERC20_PROPERTIES_NUM)):
                 address = addresses_chunk[idx]
-                info = self._process_and_create_erc20_info(
-                    output=single_output,
-                    address=address,
-                )
+                try:
+                    info = self._process_and_create_erc20_info(
+                        output=single_output,
+                        address=address,
+                    )
+                except NotERC20Conformant:
```

Comment:
shouldn't you log some warning here? If it's logged somewhere else, then this should just use suppress and not try/except continue.